plugins/generator-1.20.1/forge-1.20.1/templates/overlay.java.ftl

```diff
@@ -91,16 +90,32 @@ package ${package}.client.screens;
             </#list>
 
             <#list data.getComponentsOfType("Label") as component>
-                <#assign x = component.x - 213>
-                <#assign y = component.y - 120>
                     <#if hasProcedure(component.displayCondition)>
                         if (<@procedureOBJToConditionCode component.displayCondition/>)
                     </#if>
-                    event.getGuiGraphics().drawString(Minecraft.getInstance().font,
-                        <#if hasProcedure(component.text)><@procedureOBJToStringCode component.text/><#else>Component.translatable("gui.${modid}.${registryname}.${component.getName()}")</#if>,
-                        posX + ${x}, posY + ${y}, ${component.color.getRGB()}, false);
+                    <#if component.getAnchorPoint().name() == "TOP_LEFT">

```

Comment:
If you will re-use the same code for other components, I suggest you move it to another util file or create a macro you just put at the end of the file if it will only be used inside this specific code template (if it's used in other code templates, a util file should be used).

Reply:
LGTM